opensuse_buildservice/notqmail-rpmlintrc

```diff
@@ -0,0 +1,21 @@
+addFilter(".* devel-file-in-non-devel-package")
+addFilter(".* unversioned-explicit-self-provides")
+addFilter(".* non-standard-gid")
+addFilter(".* is not allowed anymore in FHS 2.2.")
+addFilter(".* invalid-license .*")
+addFilter(".* permissions-world-writable .* /var/qmail/queue/lock/trigger is packaged with world writable permissions")
+addFilter("W: suse-filelist-forbidden-fhs23 /var/qmail is not allowed in FHS 2.3")
+addFilter("W: permissions-file-setuid-bit")
+addFilter("W: permissions-directory-setuid-bit .* is packaged with setuid/setgid bits .*")
+addFilter("W: permissions-missing-verifyscript missing %verify_permissions -e /var/qmail/bin/.*")
+addFilter("W: file-not-in-%lang .*")
+addFilter("W: non-standard-dir-in-var .*")
+addFilter("W: non-standard-uid")
+addFilter("W: source-or-patch-not-bzipped .*")
+addFilter("W: source-or-patch-not-compressed bz2 .*")
+addFilter("W: macro-in-comment .*")
+addFilter("W: non-position-independent-executable .*")
+addFilter("W: zero-length /var/qmail/queue/lock/sendmutex")
+setBadness('permissions-unauthorized-file', 0)
+setBadness('permissions-file-setuid-bit', 0)
+setBadness('non-position-independent-executable', 0)
```

Comment:
Can we just add -fPIC to conf-cc, and maybe -fPIE to conf-load?

Reply:
Yes for the non-position-independent-executable. Will make that change.

Reply:
conf-cc and conf-ld are now modified in the spec and rules file